src/translations/components/MessageBox.ts

```diff
@@ -2,10 +2,21 @@ import {translation as _} from '../commons';
 
 const MessageBoxTexts = {
   a11yHintActionPrefix: _('Aktiver for å ', 'Activate to ', 'Aktiver for å'),
-  a11yHintUrlPrefix: _('Aktiver for å åpne lenke ', 'Activate to open link ', 'Aktiver for å opne lenke '),
+  a11yHintUrlPrefix: _(
+    'Aktiver for å åpne lenke ',
+    'Activate to open link ',
+    'Aktiver for å opne lenke ',
+  ),
   dismiss: {
     allyLabel: _('Lukk melding', 'Close message', 'Lukk melding'),
   },
+
+  a11yCriticalityPrefix: {
+    valid: _('Vellykket', 'Success', 'Vellykka'),
+    info: _('Info', 'Info', 'Info'),
+    warning: _('Advarsel', 'Warning', 'Advarsel'),
+    error: _('Feilmelding', 'Error', 'Feilmelding'),
+  },
```

Comment:
Coming to think of it, we should probably reuse the `messageTypes` in `dictionary.ts` instead?

Reply:
Hmm. Didn't think of those! Good point. Also see this is used by a different message component (🤔(

Reply:
That also explains why this is a regression. Prefix is present in other component, but not this. Anyways. fixed